File: assignment1/program2.cs
using System;

class Program
{
    static void Main(string[] args)
    {
        Task2();
    }

    static void Task2()
    {
        int n = 100;
        int evenCount = 0;
        Random rand = new Random();
        int[] a = new int[n];
        for (int i = 0; i < n; i++)
        {
            a[i] = rand.Next(1,10000000);
        }
        for (int i = 0; i < n; i++)
        {
            if (a[i] % 2 == 0)
            {
                evenCount++;
            }
        }
        Console.WriteLine("Even Count: " + evenCount);
    }
}

File: assignment1/program2.swift
import Foundation

func Task2(n : Int){
    var a : [Int] = []
    var evenCount = 0
    for _ in 0..<n{
        a.append(Int.random(in: 1...1000000))
    }
    for i in 0..<n{
        if(a[i]%2 == 0){
            evenCount += 1
        }
    }
    print("Even Count: \(evenCount)")
}

Task2(n: 1000)